Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -152,24 +155,31 @@ private void GenerateCorrelationIdAndAddToDictionary(string ikey, string appId)
         {
             try
             {
+#if !NETCORE
                 SdkInternalOperationsMonitor.Enter();
-
+#endif
                 Uri appIdEndpoint = this.GetAppIdEndPointUri(instrumentationKey);
-
+#if !NETCORE
                 WebRequest request = WebRequest.Create(appIdEndpoint);
                 request.Method = "GET";
 
                 using (HttpWebResponse response = (HttpWebResponse)await request.GetResponseAsync().ConfigureAwait(false))
+                using (StreamReader reader = new StreamReader(response.GetResponseStream()))
```

Comment:
It is correct the way it is, successive usings can stack and don't indent again if they are sharing a single set of braces (which these are).